plutus-core-interpreter/src/Language/PlutusCore/Interpreter/CekMachine.hs

```diff
@@ -224,6 +225,11 @@ readKnownCek means term = do
             appErr = ConstAppMachineError $ UnreadableBuiltinConstAppError term err
         EvaluationSuccess (Right x)  -> Right $ EvaluationSuccess x
 
+-- | Run a program using the CEK machine.
+-- Calls 'evaluateCekCatch' under the hood.
+runCekCatch :: DynamicBuiltinNameMeanings -> Program TyName Name () -> Either CekMachineException EvaluationResultDef
+runCekCatch means (Program _ _ term) = evaluateCekCatch means term
```

Comment:
If this is now a default, then I'd rename `runCek` to `unsafeRunCek` and `runCekCatch` to `runCek`.

Reply:
Hm yes.